src/platform/datapath_raw_socket_win.c

```diff
@@ -61,17 +61,6 @@ CxPlatRemoveSocket(
 {
     CxPlatRwLockAcquireExclusive(&Pool->Lock);
     CxPlatHashtableRemove(&Pool->Sockets, &Socket->Entry, NULL);
-
```

Comment:
I don't understand how we can completely remove the use of aux socket. For QTIP, we need a bound TCP socket so that the OS won't bind to the same local address.

Reply:
I think the AuxSocket is covered by normal socket

Reply:
How does normal socket cover that? Do we bind a TCP socket at all?